components/brave_ads/core/internal/deprecated/confirmations/confirmation_state_manager.cc

```diff
@@ -107,6 +107,9 @@ std::string ConfirmationStateManager::ToJson() {
 bool ConfirmationStateManager::FromJson(const std::string& json) {
   const std::optional<base::Value::Dict> dict =
       base::JSONReader::ReadDict(json);
+  confirmation_tokens_.RemoveAll();
```

Comment:
maybe we can move it to a separate function like ResetTokens()

Reply:
Because this code is being deprecated, i.e. we are moving away from this legacy JSON state I would prefer to leave as is for now.